clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -289,6 +430,7 @@ private void releasePool(String poolType, String poolName, ResourcePageSource re
     ServerSideConfiguration.Pool pool = resourcePageSource.getPool();
     OffHeapResource source = services.getService(OffHeapResourceIdentifier.identifier(pool.getServerResource()));
     if (source != null) {
+      unRegisterPoolStatistics(resourcePageSource);
```

Comment:
I actually call unRegisterPoolStatistics(resourcePageSource); in EhcacheStateServiceImpl.releasePool(...), which is called by EhcacheStateServiceImpl.destroy().  Is there a reason why I can't do it this way and need to do it from EhcacheActiveEntity.destroy()?